### PR TITLE
update(find-project-root): missing constants

### PR DESCRIPTION
**Pull request by peterblazejewicz**
- `MAX_DEPTH`
- `MARKERS`

https://github.com/kirstein/find-project-root/blob/master/index.js#L32-L33

Thanks!

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

### CODE REVIEW

File: types/find-project-root/index.d.ts [hunk 1]
```diff
@@ -22,6 +19,9 @@ declare namespace findProjectRoot {
          */
         markers?: string[];
     }
+
+    const MAX_DEPTH: 6;
```
Comment (ikatyang) [inline types/find-project-root/index.d.ts:23]:
```suggestion
    const MAX_DEPTH: 9;
```

Comment (peterblazejewicz) [reply types/find-project-root/index.d.ts:23]: Excellent, thx!

File: types/find-project-root/find-project-root-tests.ts [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
 import findProjectRoot = require("find-project-root");
 
-const root: string | null = findProjectRoot(process.cwd(), {
-    maxDepth: 12
+const root: string | null = findProjectRoot("./", {
+    maxDepth: 12,
+});
+
+findProjectRoot.MARKERS; // $ExpectType [".git", ".hg"]
+findProjectRoot.MAX_DEPTH; // $ExpectType 6
```
Comment (peterblazejewicz) [inline types/find-project-root/find-project-root-tests.ts:8]:
```suggestion
findProjectRoot.MAX_DEPTH; // $ExpectType 9
```

**Review by ikatyang — APPROVED** [review]

**Comment by peterblazejewicz** [comment]
Ready to merge 💋

**Pull request merged**